### PR TITLE
Fix inhalers [RDY]

### LINKED ISSUE #16223: inhaler_stimgas does not spawn anywhere and cannot be crafted
inhaler_stimgas does not spawn anywhere, cannot be crafted and is missing a "use_action" field. There are also no notes saying its a debug item or yet to be implemented. Should we make this item useful? or get rid of it? It would be kinda cool to have a CDDA take on jet from Fallout.

### LINKED ISSUE #16324: Inhaler is grossly overweight.
So, I just updated and reloaded a game for the first time in about two weeks, to find my asthmatic character is carrying an inhaler with a volume of 2, weight of 4.3, and 194 charges of albuterol. After my initial freakout, I went to WISH for a new inhaler out of curiosity and found they are, by default, volume 1 and weight 2.2 at 100 charges. 

While I'm going to assume the over-charge that lead to the weight gain and volume increase is a separate bug from reloading a comparatively old save, the weight of 2.2 lbs seems to be FAR too high to be realistic. I'd say the same for the volume of 1 (which I've heard described in IRC as "fist sized"), but that's less severe of an issue I imagine. 

As it stands, I'd suggest dropping the weight of a full inhaler to at most maybe 0.2 or 0.3 lbs, based on estimates I've gotten from nearby nurses and friends online who use rescue inhalers. I'll edit this with an exact answer if I can finally get Google to cooperate and not tell me all about "steroids and weight loss" when trying to find an answer to this. 

EDIT: I have found one answer thus far, [here](https://healthunlocked.com/blf/posts/130770168/has-anyone-got-a-way-of-tracking-how-much-is-left-in-a-ventolin-inhaler) listing the weight of a full inhaler canister (without the inhaler body) at 34 grams or ~0.07 lbs. Will update later.

### PR DESCRIPTION
**Pull request by mugling**
Inhalers become tools, especially because `inhaler (60) + inhaler (20) != inhaler (80)`

Fixes #16324
Fixes #16223

### CODE REVIEW

**Comment by illi-kun** [comment]
Is your fix for inhaler applicable to issue described in #10133?

**Comment by mugling** [comment]
Yes actually. If you want to review this PR now I'll include it here?

**Comment by illi-kun** [comment]
Sure, include it here and I'll review them all together.

**Comment by mugling** [comment]
Actually change of plan, ecigs are very different so I'll go with a separate PR for that

File: src/iuse.cpp [hunk 1]
```diff
@@ -954,11 +954,12 @@ int iuse::flusleep(player *p, item *it, bool, const tripoint& )
...
 int iuse::inhaler(player *p, item *it, bool, const tripoint& )
 {
-    p->remove_effect( effect_asthma);
-    p->add_msg_if_player(m_neutral, _("You take a puff from your inhaler."));
-    if (one_in(50)) {  // adverse reaction
-        p->add_msg_if_player(m_bad, _("Your heart begins to race."));
-        p->mod_fatigue(-10);
+    p->add_msg_if_player( m_neutral, _( "You take a puff from your inhaler." ) );
+    if( !p->remove_effect( effect_asthma) ) {
+        p->mod_fatigue( -10 ); // if we don't have asthma can be used as stimulant
```
Comment (illi-kun) [inline src/iuse.cpp]: If I read it correctly, you significantly increase its stimulating effect from some minor chance of fatigue reduction (```one_in(50)```) to guaranteed one. I'm able to decrease level of fatigue from "dead tired" to OK condition by 30 puffs. I'm not sure it's intended/good change.

We can add some more significant penalties for overdosing, including hallucinations, vomiting, and so on.

Comment (mugling) [inline src/iuse.cpp]: The `one_(50)` change was intentional (spamming 50 keypresses to get an effect isn't useful) however it looks like it became a little too good.

I think `shakes` is quite an appropriate effect. How about reducing the `fatigue` bonus from 10 to 2 and keeping the same risk (1:20) of an adverse effect (`shakes`)?

**Comment by illi-kun** [comment]
My fear is that we replace one spamming mechanics by another one.

Drop from 10 bonus points to 3 pts. looks acceptable for me (if nobody proposes less-spammy mechanics).

**Comment by mugling** [comment]
Lets go with that for now, see 4bededb

**Pull request merged**